.gitmodules

```diff
@@ -43,4 +43,4 @@
 	url = https://github.com/Shillaker/wasi-libc
 [submodule "third-party/wamr"]
 	path = third-party/wamr
-	url = https://github.com/J-Heinemann/wasm-micro-runtime 
+	url = https://github.com/bytecodealliance/wasm-micro-runtime 
```

Comment:
I couldn't see any differences between your fork and the main repo. Obviously feel free to switch back in your own fork

Reply:
Yes, you're right, I changed a CMakeFile in WAMR and reverted these changes later (https://github.com/J-Heinemann/wasm-micro-runtime/commit/94f5108a062e2a315567f7ed60d1eadee34ceb22) because I changed the Faasm CMakeFile.